src/support/loader.asm

```diff
@@ -157,7 +158,7 @@ bit_bsx:
 cmp r4,#0x30 // Check if BSX
 jp nz, check_pal
 log_string("Using BSX")
-// It's BSX
+jp expansion_core// It's BSX
```

Comment:
This doesn't do anything

Reply:
hey agg, can you refuse / cancel the whole pull request ? ive tried from my end but cant seem to find an option to delete/cancel. I have changed a few things.

also i dont think we will ever get the pocket core to load bsx roms like mister does as mister detects bs roms via the file extension ".bs"
https://github.com/MiSTer-devel/Main_MiSTer/blob/222d61776f3cc534f1e48ec1a6e31a3bc1588752/user_io.cpp#L2506

It also detects the bios which isnt a problem(we can do that)  also the header is modified on load on some roms as they were playable only at set times. So the way i load bsx is detect the bsx bios header which concatenated with each rom